libs/clients/signature-collection/src/lib/types/collection.dto.ts

```diff
@@ -1,67 +1,32 @@
 import { Area, mapArea } from './area.dto'
-import {
-  MedmaelasofnunDTO,
-  MedmaelasofnunExtendedDTO,
-} from '../../../gen/fetch'
+import { MedmaelasofnunExtendedDTO } from '../../../gen/fetch'
 import { logger } from '@island.is/logging'
 import { Candidate, mapCandidate } from './candidate.dto'
 
-export interface CollectionInfo {
-  id: number
+export interface Collection {
+  id: string
   startTime: Date
   endTime: Date
   isActive: boolean
   isPresidential: boolean
   isSignatureCollection: boolean
-}
-export interface Collection extends Omit<CollectionInfo, 'id'> {
-  id: string
   name: string
-  startTime: Date
-  endTime: Date
   areas: Area[]
   candidates: Candidate[]
 }
 
-export function mapCollectionInfo(
-  collection: MedmaelasofnunDTO,
-): CollectionInfo | null {
-  const {
-    id: id,
-    sofnunStart: startTime,
-    sofnunEnd: endTime,
-    kosning,
-  } = collection
-
-  if (id == null || startTime == null || endTime == null) {
-    logger.warn(
-      'Received partial collection information from the national registry.',
-      collection,
-    )
-    return null
-  }
-  return {
-    id,
-    startTime,
-    endTime,
-    isActive: startTime < new Date() && endTime > new Date(),
-    isPresidential: collection.kosningTegund == 'Forsetakosning',
-    isSignatureCollection: kosning?.erMedmaelakosning ?? false,
-  }
-}
-
 export function mapCollection(
   collection: MedmaelasofnunExtendedDTO,
 ): Collection {
   const {
-    id: id,
+    id,
     sofnunStart: startTime,
     sofnunEnd: endTime,
     svaedi: areas,
     frambodList: candidates,
     kosning,
   } = collection
-  if (id == null || startTime == null || endTime == null || areas == null) {
+  if (!id || startTime == null || endTime == null || areas == null) {
     logger.warn(
       'Received partial collection information from the national registry.',
       collection,
```

Comment:
What happens if `id` is set and all other values, `startTime`, `endTime` and `areas` are `undefined`?

Should these be tested against with `!startTime || !endTime || ...etc`?

Reply:
Ah, double equals. Nevermind.
However, should we avoid double equals?

Reply:
Because, doesn't this feel yucky? :laughing: 
```ts
> startTime = undefined
undefined
> startTime == null
true
> startTime === null
false
```